tests/base.py

```diff
@@ -217,6 +236,11 @@ def wrapper(self, url, params=NoDefault, **kw):
     patch_json_api = json_api_method('PATCH')
     delete_json_api = json_api_method('DELETE')
 
+    bulk_post_json_api = bulk_json_api_method('POST')
```

Comment:
Rather than creating a new set of methods, it might make sense to add a `bulk` parameter to the `json_api_methods`. e.g.

```python
app.put_json_api(url, {...}, bulk=True)
```